src/Core/Components/Dialog/FluentDialogHeader.razor.cs

```diff
@@ -34,6 +33,18 @@ public partial class FluentDialogHeader : FluentComponentBase
     [Parameter]
     public bool? ShowDismiss { get; set; }
 
+    /// <summary>
+    /// When true, shows the "Close" button tooltip in the header.
+    /// </summary>
+    [Parameter]
+    public bool? ShowDismissTooltip { get; set; } = true;
+
+    /// <summary>
+    /// When 0, the "Close" button tooltip will not be displayed when the dialog is open.
```

Comment:
Can  you update this comment based on the ["official" doc](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex)?